tests/run_test.sh

```
@@ -1,13 +1,23 @@
+    if [ -n "$SQL_PID" ]; then
+        kill $SQL_PID 2>/dev/null || true
+        wait $SQL_PID 2>/dev/null || true
+    fi
+}
+
+# Trap exit, interrupt and error signals
+trap cleanup EXIT INT ERR
+
 rm -rf ../test_data || true
 mkdir -p ../test_data
 cd ../build
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)